includes/Modules/AdSense/Ad_Blocking_Recovery_Web_Tag.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\AdSense\Ad_Blocking_Recovery_Web_Tag
+ *
+ * @package   Google\Site_Kit\Modules\AdSense
+ * @copyright 2023 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\AdSense;
+
+use Google\Site_Kit\Core\Modules\Tags\Module_Web_Tag;
+use Google\Site_Kit\Core\Util\Method_Proxy_Trait;
+use Google\Site_Kit\Core\Tags\Tag_With_DNS_Prefetch_Trait;
+
+/**
+ * Class for Web tag.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Ad_Blocking_Recovery_Web_Tag extends Module_Web_Tag {
+
+	use Method_Proxy_Trait, Tag_With_DNS_Prefetch_Trait;
```

Comment:
Nit: please put traits on their own line.
```suggestion
	use Method_Proxy_Trait
	use Tag_With_DNS_Prefetch_Trait;
```